### DISCUSSION TITLE
Realtime Events in Kotlin

### DISCUSSION THREAD

**Discussion by billypap1** [supabase/supabase#6068] [Questions] [answered]
Hello, I want to integrate Supabase to my Kotlin app.
I found these libraries -> postgrest-kt - gotrue-kt but none of them contain docs for Realtime Features.
Are there any other library that support listen for updates/Realtime Features in order to use it in my Kotlin app?

**Comment by w3b6x9** [comment, marked as answer]
@billypap1 I don't know of any `realtime-kt` client libs but if you find one or create one please let us know!

**Reply by ebabel** [reply]
https://github.com/supabase-community/supabase-kt/wiki/Realtime

**Reply by Zulfiddinovich** [reply]
link is updated, use this:
https://github.com/supabase-community/supabase-kt/tree/master/Realtime
